app/graphql/mutations/post_mutation.rb

```diff
@@ -67,5 +67,33 @@ module Mutations
         end
       end
     end
+
+    field :deletePost, Types::PostType do
+      argument :id, types.ID, 'The ID of the post you want to delete'
+
+      type types.ID
+
+      resolve ->(_obj, args, ctx) do
+        break unless args[:id]
+
+        current_user = ctx[:current_user]
+
+        if current_user.blank?
+          raise GraphQL::ExecutionError.new("Authentication required")
+        end
+
+        post = Post.find(args[:id])
+
+        break unless post.sender == current_user || current_user.admin_of?(post.team) || current_user.admin?
+
+        break if post.created_at < Settings.deletable_time_for_post && post.sender == current_user
+
+        if post.destroy
+          return post.id
+        else
+          GraphQL::ExecutionError.new(post.errors.full_messages.join(', '))
```

Comment:
[On this line](https://github.com/kabisa/kudo-o-matic/pull/478/files#diff-b9dfc89fe45f11475a16c9b1c28148d2R82) you raise an error, here you return it. Is this on purpose?

Reply:
This should `raise` as well. 
```suggestion
          raise GraphQL::ExecutionError.new(post.errors.full_messages.join(', '))
```

Reply:
Did you add this change or not? Github shows me a suggested change only.